### PR TITLE
Improvement in FAQ.

### PR DESCRIPTION
**Pull request by quamilek**
Added information about synchronization users from LDAP.

### CODE REVIEW

**Comment by vi4m** [comment]
LGTM

File: doc/faq.rst [hunk 1]
```diff
@@ -114,6 +114,11 @@ You can define users filter, if you don't want to import all users to ralph::
 
 In case of OpenDJ please use ``isMemberOf`` instead of ``memberOf``.
 
+To synchronize user list you must run command::
+
+    ralph ldap_sync
+
+It may take a while, be patient. Users are synced in packs of 100.
```
Comment (vi4m) [inline doc/faq.rst]: During the process, script will report progress on every 100-th item loaded.

**Comment by ghost** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**